src/middlewared/middlewared/utils/__init__.py

```diff
@@ -442,3 +442,21 @@ def get_service(self, name):
 
     def get_services(self):
         return self._services
+
```

Comment:
This file is too big already, please put all new utils to separate modules

Reply:
Make sense. Now it is isolated.